lib/pi_piper/bcm2835.rb

```diff
@@ -1,5 +1,15 @@
-require "pi_piper/bcm2835/version"
+require 'pi_piper/bcm2835/version'
+require 'pi_piper/bcm2835/pin'
+require 'pi_piper/bcm2835/spi'
+require 'pi_piper/bcm2835/i2c'
+require 'pi_piper/bcm2835/driver'
 
 module PiPiper
-  autoload PiPiper::Bcm2835, "pi_piper/bcm2835/bcm2835"
+  module Bcm2835
+    class << self
+      def driver
+        @driver ||= PiPiper::Bcm2835::Driver.new
+      end
```

Comment:
Yeah I thought we would do something like this in the user code:

```ruby
require 'pi_piper'
require 'pi_piper-sysfs' or require 'pi_piper-bcm2835'

PiPiper.driver = PiPiper::Sysfs.driver || PiPiper::Bcm2835.driver
# ... do stuff ...
```
And yes, only 1 driver would be loaded for `PiPiper` at any given moment.
Is that what you meant, or something different?